learning/tour-of-beam/terraform-v2/README.md

```diff
@@ -0,0 +1,60 @@
+# [Tour of Beam] Deploying backend infrastructure as GCP Cloud Function with GCP, Terraform and Cloud Build
+This tutorial explains how to create Tour of Beam backend infrastructure as a GCP Cloud Functions using Infrastructure as a Code (IaC) and GitOps methodoliges.
+
+
+### Deployment
+Prerequisites:
+- Active GCP project with billing enabled
+- Existing bucket to store Terraform state (name to be declared in backend.tf)
+- Existing service account with the following roles:
+    - Cloud Functions Admin
+    - Project IAM Admin
+    - Service Account Admin
+    - Service Account User
+    - Storage Admin
+    - Storage Object Admin
+- Exported JSON Key of service account above
+
+#### Configuring your environment
+
+Steps below will:
+1. Enable required APIs for the project
+2. Create service account and assign required IAM roles to it (service account to run the function with)
```

Comment:
Usually we deploy infrastructure under some Service Account already, maybe we don't create another?

Reply:
I think that Ruslan is correct. We need to have a separate SA for cloud functions to follow the minimum priviledge principle